### DISCUSSION TITLE
Somthing wrong with PrimeNg Dropdown component

### DISCUSSION THREAD

**Discussion by pouyababaie** [primefaces/community#3186] [PrimeNG] [answered]
I was looking for the documentation , and i guess it has been removed or they missed somthing . I'm opening this discusion for future missed docs. feel free to inform me if i'm wrong

**Comment by mazerdeci** [comment, marked as answer]
Hi,

Some components are renamed. New name is Select.

[Related Page](https://primeng.org/guides/migration#compatible)

**Reply by pouyababaie** [reply]
thank you for your information.
I totally missed the migration to V18 part of the website.

**Comment by ibandim123** [comment]
https://v17.primeng.org/dropdown#import

Dropdown component still there, but virtual scroll property be broken 
![{6A10B6F8-135E-43BB-AD22-4934DC24A7EC}](https://github.com/user-attachments/assets/b9aac52e-c79c-42fa-bc74-59eba92c5c6e)
